website/docusaurus.config.js

```diff
@@ -64,6 +64,7 @@ module.exports = {
         fromExtensions: ['html'],
       },
     ],
+    /*
```

Comment:
is this the reason it's a draft?

Reply:
In part, but also because it is using code that is not even merged to master :) 

Will finish this soon

Reply:
just updated the PR, fixing the pwa issue.

We'll merge the pr next week and release it so you can merge this one when we have a stable version (at some point Jest site should use a stable version 😅 )